packaging/MacSDK/fsharp.py

```diff
@@ -2,15 +2,13 @@ class FsharpPackage(GitHubTarballPackage):
     def __init__(self):
         GitHubTarballPackage.__init__(self,
             'fsharp', 'fsharp',
-            '4.1.34',
-            '662492595a63dffff8fac84939614743fd6d34f9',
+            '4.5.0',
+            '3de387432de8d11a89f99d1af87aa9ce194fe21b',
             configure='./configure --prefix="%{package_prefix}"',
             override_properties={ 'make': 'make' })
 
         self.extra_stage_files = ['lib/mono/xbuild/Microsoft/VisualStudio/v/FSharp/Microsoft.FSharp.Targets']
         self.sources.extend(['patches/fsharp-portable-pdb.patch',
-                             'patches/fsharp-string-switchName.patch',
-                             'patches/fsharp-path-overloads.patch',
                              'patches/fsharp-debug-pinvoke-fix.patch'])
```

Comment:
@nosami btw do you think we can remove those two remaining patches as well now?

Reply:
No. Not yet. I will try and get them upstreamed.